misk-crypto/src/test/resources/test-config.yaml

```diff
@@ -0,0 +1,4 @@
+aws_kms_key_alias: "master_key_name"
```

Comment:
worth mentioning here or somewhere else how you generated this?

Reply:
Or in the readme?